content/blog/2021/02/2021-02-08-docker-base-os-upgrade.adoc

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "Docker image updates"
+tags:
+- announcement
+- docker
+- platform-sig
+author: markewaite
+opengraph:
+  image: /images/docker/dockerJenkins-updated-os-2021-02.png
+---
+
+Beginning with Jenkins 2.279 and Jenkins 2.263.4, the Jenkins project is upgrading the base operating system and Java version used in the `jenkins/jenkins:latest` and `jenkins/jenkins:lts` images.
```

Comment:
Java 11 also?

Reply:
The Java 11 images are already running Debian 10.

For Java 11:
```
$ docker run --rm -i -t jenkins/jenkins:lts-jdk11 cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 10 (buster)"
NAME="Debian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

For Java 8:

```
$ docker run --rm -i -t jenkins/jenkins:lts cat /etc/os-release
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
VERSION_CODENAME=stretch
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```